.verb.md

```diff
@@ -45,28 +33,9 @@ strip(commentString, { preserveNewlines: true });
 {%= include("author") %}
 
 ## License
-{%= copyright() %}
+{%= copyright({linkify: true, start: 2014}) %}
 {%= license %}
 
 ***
 
 {%= include("footer") %}
-
-
-```js
-//example.strip
-console.log(strip("foo // this is a comment\n/* me too */"));
-//=> 'foo'
-```
```

Comment:
fwiw I use a middleware to merge these examples into the code comments from `index.js`. makes the code comments cleaner. no big deal though